### PR TITLE
chore: add docs page around js client stable id

### PR DESCRIPTION
**Pull request by daniel-statsig**

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://66eb711ba28283d82bffde47--cozy-fox-0defec.netlify.app

File: docs/client/javascript-mono/StableIdConsistency.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+StableID is a concept in Statsig that allows us to have a consistent identifier for a single device. 
+This allows us to run [experiments on logged out](/guides/first-device-level-experiment) users (Users without a User ID) 
+as well as target gates on the device level rather than the user level.
+
+
+## How it works
+
+The first time you initialize the SDK, we will check if you have a StableID set in local storage. If not, we will generate a new StableID and set it in local storage. 
+Then the next time you initialize the SDK, we will check for the StableID in local storage and reuse it.
```
Comment (tore-statsig) [inline docs/client/javascript-mono/StableIdConsistency.mdx:28]: We need to note somehwere:

local storage is only accessible per subdomain, and not accessible on the server side.  If you need an identifier across multiple subdomains or on your backend for identifying anonymous users, you need to manage a cookie yourself.  The sdk does not use any cookies

File: docs/client/javascript-mono/StableIdConsistency.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+:::
+
+
+
+## Keeping StableID Consistent
+
+If you have a backend service running a Statsig SDK as well as using the Statsig client in your frontend, 
+you may need to have the same StableID accessible in both environments.
+
+It is highly dependent on your setup, but usually you can send the StableID up with any requests from your frontend.
```
Comment (tore-statsig) [inline docs/client/javascript-mono/StableIdConsistency.mdx:100]: they wont be able to send the statsig stableid - if they are doing this they need to manage it themselves

**Review by tore-statsig — COMMENTED** [review]
should also note that all metrics will be available for stableID and userID so you can see logged in user counts and metrics and logged out

We should also note that we do not alias, we track them independently

**Comment by daniel-statsig** [comment]
> should also note that all metrics will be available for stableID and userID so you can see logged in user counts and metrics and logged out
> 
> We should also note that we do not alias, we track them independently

I don't follow what this means. I don't actually know how the metrics side of this works

File: docs/client/javascript-mono/StableIdConsistency.mdx (hunk 2 above)
Comment (daniel-statsig) [reply docs/client/javascript-mono/StableIdConsistency.mdx:100]: Ok, updated

File: docs/client/javascript-mono/StableIdConsistency.mdx (hunk 1 above)
Comment (daniel-statsig) [reply docs/client/javascript-mono/StableIdConsistency.mdx:28]: updated

**Pull request merged**